ci-operator/step-registry/hypershift/openstack/e2e/hypershift-openstack-e2e-workflow.yaml

```diff
@@ -0,0 +1,27 @@
+workflow:
+  as: hypershift-openstack-e2e
+  documentation: |-
+    This workflow is a fork from `hypershift-aws-e2e-nested-workflow`.
+    It's deploying a management cluster on AWS and then prepare
```

Comment:
The workflow is doing a more than what is described here (we only document the `pre` step). Perhaps we could expand a bit and explain what the `test` step is doing at least.

Reply:
done